src/app/services/calendar.service.ts

```diff
@@ -0,0 +1,37 @@
+import { Injectable } from '@angular/core';
+import { TranslateService } from '@ngx-translate/core';
+import { Option } from '../interfaces/option.interface';
+
+@Injectable({ providedIn: 'root' })
+export class CalendarService {
+  constructor(
+    private translate: TranslateService,
+  ) {}
+
+  getCalendarOptions(): Option[] {
+    return [
+      {
+        label: this.translate.instant('Monday'),
+        value: 1,
+      }, {
+        label: this.translate.instant('Tuesday'),
+        value: 2,
+      }, {
+        label: this.translate.instant('Wednesday'),
+        value: 3,
+      }, {
+        label: this.translate.instant('Thursday'),
+        value: 4,
+      }, {
+        label: this.translate.instant('Friday'),
+        value: 5,
+      }, {
+        label: this.translate.instant('Saturday'),
+        value: 6,
+      }, {
+        label: this.translate.instant('Sunday'),
+        value: 7,
+      },
+    ];
+  }
```

Comment:
- [ ] Define enum type for Weekday.
```
export enum Weekday {
  Mon,
  Tue,
  Wed,
  ...
}
```
- [ ] Use `getWeekdayOptions` instead of `getCalendarOptions` for extension
- [ ] Replace values to `value: Weekday.Mon`